tests/message_ut/message_ut.c

```diff
@@ -114,11 +123,8 @@ TEST_FUNCTION(message_create_succeeds)
 TEST_FUNCTION(message_create_2_times_yields_2_different_message_instances)
```

Comment:
Agree, I will add all the tests as a separate PR, I am simply converting now to umock_c.

---
In reply to: [104191365](https://github.com/Azure/azure-uamqp-c/pull/28#discussion_r104191365) [](ancestors = 104191365)